src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubTriggerAttributeBindingProvider.cs

```diff
@@ -44,20 +44,19 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
                 return Task.FromResult<ITriggerBinding>(null);
             }
 
-            string eventHubName = attribute.EventHubName;
-            string resolvedEventHubName = _nameResolver.ResolveWholeString(eventHubName);
+            string resolvedEventHubName = _nameResolver.ResolveWholeString(attribute.EventHubName);
 
-            string consumerGroup = attribute.ConsumerGroup;
-            if (consumerGroup == null)
-            {
-                consumerGroup = Microsoft.ServiceBus.Messaging.EventHubConsumerGroup.DefaultGroupName;
-            }
+            string consumerGroup = attribute.ConsumerGroup ?? EventHubConsumerGroup.DefaultGroupName;
             string resolvedConsumerGroup = _nameResolver.ResolveWholeString(consumerGroup);
 
+            if (!string.IsNullOrWhiteSpace(attribute.Connection))
```

Comment:
>if (!string.IsNullOrWhiteSpace(attribute.Connection)) [](start = 12, length = 53)

This is what I was looking for! So this is the key change for triggering, right?